pyshelf/bucket_update/search_updater.py

```diff
@@ -78,7 +78,9 @@ def add_artifact_metadata(self, path, bulk_update):
         """
         identity = self.container.resource_identity_factory \
             .from_cloud_identifier(path)
-        updater = ArtifactMetadataUpdater(self.bucket_container, identity)
```

Comment:
why put this on separate line?  If you are going to put it on separate lines could you at least use indent like

```
updater = ArtifactMetadataUpdater(
            self.bucket_container,
            identity
)
```